### PR TITLE
New release process using docker, circleci and a centralized building tool

### PR DESCRIPTION
**Pull request by sdurrheimer**

### CODE REVIEW

File: .promu.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        - netbsd/amd64
+        - netbsd/386
+        - dragonfly/amd64
+        - linux/arm
+        - linux/arm64
+        - freebsd/arm
+        - openbsd/arm
+        - netbsd/arm
+        - linux/ppc64
+        - linux/ppc64le
```
Comment (sdurrheimer) [inline .promu.yml]: @SuperQ Let me know which platforms you want to support, I guess mysqld doesn't run on all them.

Comment (SuperQ) [inline .promu.yml]: MySQL's platform support is pretty good.  The officially supported by oracle list is fairly short, but the source will compile to pretty much anything *NIXy.

Comment (sdurrheimer) [inline .promu.yml]: So we keep the full list of platforms supported by `promu` ?

Comment (SuperQ) [inline .promu.yml]: Yea, full list is fine.

**Comment by SuperQ** [comment]
Any reason we don't switch to go 1.6?

**Comment by sdurrheimer** [comment]
The switch to `go 1.6.2` has not yet been decided for all our components.

If you're OK about switch `mysqld_exporter` to `go 1.6.2`, I can do it.

**Comment by SuperQ** [comment]
Maybe easier to leave at 1.5 for now, I can start testing a 1.6 build for the next release.

**Comment by sdurrheimer** [comment]
Ready for the review

**Comment by SuperQ** [comment]
:+1: I'm wondering if we should call this 0.8.1 since it's technically a new release.

**Comment by SuperQ** [comment]
@juliusv :shipit:?

**Comment by sdurrheimer** [comment]
I'm putting this into a waiting state, we are currently refactoring the version information package for our components. prometheus/node_exporter#236

**Comment by SuperQ** [comment]
Is this ready?

**Comment by sdurrheimer** [comment]
Yep we are good to go. I will reactivate CircleCI.

**Comment by SuperQ** [comment]
Awesome, looks good. :+1:

**Pull request merged**